dropwizard-auth/src/test/java/io/dropwizard/auth/basic/BasicAuthProviderTest.java

```diff
@@ -8,8 +8,13 @@
 import java.security.Principal;
 
 public class BasicAuthProviderTest extends AuthBaseTest<BasicAuthProviderTest.BasicAuthTestResourceConfig> {
-    public static class BasicAuthTestResourceConfig extends AuthBaseResourceConfig{
-        protected ContainerRequestFilter getAuthFilter() {
+    public static class BasicAuthTestResourceConfig extends AbstractAuthResourceConfig {
+        public BasicAuthTestResourceConfig() {
+            super();
```

Comment:
You're right. I got a bit overzealous with these `super()`s while trying to get resource injection to compile with the new class hierarchy. Will remove these.